opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -203,6 +203,7 @@ namespace detail {
         , terminal_output_ (terminal_output)
         , material_name_(0)
         , current_relaxation_(1.0)
+        , rate_converter_(fluid_, std::vector<int>(std::vector<int>(AutoDiffGrid::numCells(grid_),0)))
```

Comment:
Two comments on this line: first, there seems to be an extra vector wrapping. Second, using 0 for the region seems wrong? (I am not sure what the correct ...NUM thing is to use here though.)